app/views/avoid_postings/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<div id="c-avoid-postings">
+  <div id="a-index">
+    <h1>Avoid Postings</h1>
+
+    <%= render "search" %>
+
+    <table class="striped autofit">
+      <thead>
+      <tr>
+        <th></th>
+        <% if CurrentUser.is_staff? %>
```

Comment:
You should put artist names first, details in the middle, and everything else after.

![Screenshot 2024-07-29 102341](https://github.com/user-attachments/assets/1c971ab8-4b9d-4768-9bed-e4adc18335db)

I think the layout makes a lot more sense like that.